### ISSUE TITLE
2.4.2 M2 Report Changes in Requirements

### ISSUE THREAD

**Issue by donhitme34** [INF2001/TEAM-PROJECT-P11-3#97]

**Comment by jasonyjk** [comment]
Task title ID: #97
What is the task about? **The task involves documenting the changes made to the requirements from M1. It reflects 2 key points on Use Case Description and Non-Functional Requirement in particular sub section on Safety & Security Requirements.**
What are the goals of the task? **The goal of this task is to Integrate the updated Use Case Descriptions and Non-Functional Requirements into the project's documentation**
Success or completion criteria. **All 16 Use Case Descriptions and 5 Non-Functional Requirements in M2 Section 2 have been modified to align with the updated project requirements.**
Allocated time: **144 hours**
Start date and end date of the issue/task **20/10/2023 to 25/10/2023**
Who is the owner? **Jason**
The status of the tasks. **Completed**
What is the Priority of the task? **MEDIUM**

**Comment by jasonyjk** [comment]
In our detailed work process, we have successfully revised 16 Use Case Descriptions and 5 Non-Functional Requirements based on the feedback collected during the M1 phase. We have then incorporated these updates into the M2 Report, specifically within M2 Section 2 (Changes in Requirements), to ensure alignment and clarity, making the requirements easily understandable for all stakeholders.
The project timeline remains unaffected, and it continues to progress as per the established schedule.
No, there are no changes to the success or completion criteria of the task. The criteria remain as originally defined.
Start date 20/10/2023 end date 22/10/2023
